main.js

```diff
@@ -8,6 +8,13 @@ import {
 
 import modelHandler from './popup.js';
 
+import {
+  inputs,
```

Comment:
**Kindly work on this requested changes:**
- Add a function to validate the email input that can check your input text, if it's typed correctly in lowercase.